frontend/src/views/Landing.tsx

```diff
@@ -1,21 +1,77 @@
-import React from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
+import { ThemeConfig } from '../components/config/Theme';
+import { LandingPageContainer } from '../components/core/Container';
 import { PrimaryButtonLink, TextLink } from '../components/core/Link';
-import { LandingHeaderText } from '../components/core/Text';
+import { LandingHeaderText, LandingHeaderTitle } from '../components/core/Text';
+import { FloatingCircle, Coordinate } from '../components/special/FloatingCircle';
 
 function LandingPage() {
+  const [mousePosition, setMousePosition] = useState<Coordinate>({ x: 0, y: 0 });
+
+  const mouseMoveHandler = useCallback((e: MouseEvent) => {
+    setMousePosition({ x: e.clientX, y: e.clientY });
+  }, [setMousePosition]);
+
+  useEffect(() => {
+    window.onmousemove = mouseMoveHandler;
+  }, [mouseMoveHandler]);
+
   return (
-    <div>
+    <LandingPageContainer>
+      <FloatingCircle
+        color={ThemeConfig.colors.redCircle}
+        x={mousePosition.x}
+        y={mousePosition.y}
+        bottom={50}
+        left={-30}
+        size={4}
+      />
```

Comment:
Thoughts on moving this into a new say `layout/Circles.tsx` (maybe a better name though idk haha) file? That way we can reuse this background on different pages (if we choose to) by doing something like this in the `config/App.tsx` file: 

```jsx
<CustomRoute path="/" component={LandingPage} layout={CircleLayout} exact />
```

Reply:
That's a good idea. I will say that this background is fairly customized to this page, but others like the "Join a Room" page and "Create a Room" page may be relatively similar to the landing page design. Done!